tools/plugins/com.liferay.ide.sdk.core/src/com/liferay/ide/sdk/core/SDK.java

```diff
@@ -63,6 +63,8 @@ public static String createXMLNameValuePair( String name, String value )
 
     public static List<String> SUPPORT_SERVER_TYPES = Arrays.asList( new String[]{ "tomcat", "jboss", "glassfish", "jetty"} );
 
+    public static Map<String,Object> BUILD_PROPERTIES = null;
```

Comment:
This can not be static.  That means that for every single SDK in the whole JVM it would have teh same build.properties.  I know that we have a one-sdk per one-workspace rule, but we shouldn't program that way.  We should have another map that is static that holds a map of sdk location path -> BUILD_PROPERTIES map.